pkg/services/sqlstore/stats.go

```diff
@@ -51,14 +60,32 @@ func GetSystemStats(query *m.GetSystemStatsQuery) error {
 		SELECT COUNT(*)
 		FROM ` + dialect.Quote("alert") + `
 	  ) AS alerts,
-			(
-				SELECT COUNT(*) FROM ` + dialect.Quote("user") + ` where last_seen_at > ?
-	  ) as active_users
+		(
+			SELECT COUNT(*) FROM ` + dialect.Quote("user") + ` where last_seen_at > ?
+		) as active_users,
+		(
+			SELECT COUNT(id) FROM ` + dialect.Quote("dashboard") + ` where is_folder = ?
+		) as folders,
+		(
+			SELECT COUNT(acl.id) FROM ` + dialect.Quote("dashboard_acl") + ` as acl inner join ` + dialect.Quote("dashboard") + ` as d on d.id = acl.dashboard_id where d.is_folder = ?
+	  ) as dashboard_permissions,
+		(
+			SELECT COUNT(acl.id) FROM ` + dialect.Quote("dashboard_acl") + ` as acl inner join ` + dialect.Quote("dashboard") + ` as d on d.id = acl.dashboard_id where d.is_folder = ?
+		) as folder_permissions,
+		(
+			SELECT COUNT(id) FROM ` + dialect.Quote("dashboard_provisioning") + `
+		) as provisioned_dashboards,
+		(
+			SELECT COUNT(id) FROM ` + dialect.Quote("dashboard_snapshot") + `
+		) as snapshots,
+		(
+			SELECT COUNT(id) FROM ` + dialect.Quote("team") + `
+		) as teams
 			`
 
 	activeUserDeadlineDate := time.Now().Add(-activeUserTimeLimit)
 	var stats m.SystemStats
-	_, err := x.SQL(rawSql, activeUserDeadlineDate).Get(&stats)
+	_, err := x.SQL(rawSql, activeUserDeadlineDate, dialect.BooleanStr(true), dialect.BooleanStr(false), dialect.BooleanStr(true)).Get(&stats)
```

Comment:
I think we should declare those bool strings as variables or add them to the sql statement.

Reply:
Pushed a change to use the sql builder instead.